apis/testsuite/v1/testsuite_types.go

```diff
@@ -40,8 +41,26 @@ type TestSuiteSpec struct {
 	// schedule in cron job format for scheduled test execution
 	Schedule string `json:"schedule,omitempty"`
 
-	// execution params passed to executor
+	// DEPRECATED execution params passed to executor
 	Params map[string]string `json:"params,omitempty"`
+	// Variables are new params with secrets attached
+	Variables map[string]Variable `json:"variables,omitempty"`
+}
+
+const (
+	VariableTypeBasic  = "basic"
+	VariableTypeSecret = "secret"
+)
+
+type Variable struct {
```

Comment:
can we have them in the same file and avoid duplication? 😄

Reply:
moved it to common pkg and used in both